Dockerfile

```
@@ -1,8 +1,15 @@
-FROM alpine:3.17
-
-# install packages required to run the tests
-RUN apk add --no-cache jq coreutils
+FROM ubuntu:bionic
```

It's best to be as specific as possible

```suggestion
FROM ubuntu:bionic-20230308
```